models/stg_hubspot__ticket.yml

```diff
@@ -140,6 +114,10 @@ models:
         description: The ID of the pipeline stage.
       - name: ticket_state
         description: State of the ticket.
+      - name: ticket_pipeline_created_at
+        description: A timestamp representing when the record was created.
+      - name: ticket_pipeline_updated_at
+        description: A timestamp representing when the record was updated.
```

Comment:
looks like the ticket_pipeline_stage and ticket_pipeline fields got mixed up in here

Reply:
Yes thank you again. Updated.